content/index.html

```diff
@@ -110,7 +110,7 @@ <h5>Want to dig in?</h5>
               </div>
               <div class="grid-flex-cell-1of3 content-center">
                 <h5>Hands-on learner?</h5>
-                <a class="button button-primary" href="https://proto.school/#/tutorials">Explore ProtoSchool</a>
+                <a class="button button-primary" href="http://proto.school/#/tutorials?course=ipfs">Explore ProtoSchool</a>
```

Comment:
Any reason not to use https?

Reply:
Nice catch. I probably copy/pasted from localhost and then typed in the proto.school. Will fix before merging!